draft-ietf-cats-usecases-requirements.md

```diff
@@ -770,6 +770,134 @@ informative:
    WAN1 -> Cloud2 vCPE1 -> Cloud2 APP1
 
 
+##  Computing-Aware AI large model
+   
+   AI large model refers to a type of artificial intelligence model that 
+   is trained on massive amounts of data using deep learning techniques. 
```

Comment:
Can we have a citation here?

Reply:
Actually, it is not a direct citation, but a rewording of several citations, including:

- https://www.merantix.com/blog/the-significance-of-ai-models#:~:text=Large%20AI%20Models%2C%20also%20known,are%20used%20to%20build%20them.
- https://aws.amazon.com/cn/blogs/aws/generative-ai-with-large-language-models-new-hands-on-course-by-deeplearning-ai-and-aws/#:~:text=Generative%20AI%20is%20powered%20by,across%20many%20natural%2Dlanguage%20tasks.
- https://techcommunity.microsoft.com/t5/ai-machine-learning-blog/training-large-models-in-azure-machine-learning/ba-p/3834865

Is it ok to list several citation links?

Reply:
I would pick one of these and cite it.

Reply:
Done. Please take a look